src/services/routing.ts

```diff
@@ -39,6 +50,13 @@ export const ROUTES = {
   labels: (service: string) => prefixRoute(`${PageSlugs.explore}/service/${replaceSlash(service)}/${PageSlugs.labels}`),
 };
 
+export const SUB_ROUTES = {
+  label: (service: string, label: string) =>
+    prefixRoute(`${PageSlugs.explore}/service/${replaceSlash(service)}/${ValueSlugs.label}/${label}`),
```

Comment:
I guess we should call `replaceSlash` on `label` too?

```suggestion
    prefixRoute(`${PageSlugs.explore}/service/${replaceSlash(service)}/${ValueSlugs.label}/${replaceSlash(label)}`),
```

Reply:
Ah, it's label name. Yea, sure! Good call!